### DISCUSSION TITLE
找地产相关方的人聊了一圈，现在的浅见。

### DISCUSSION THREAD

**Discussion by cc2218** [WeNeedHome/SummaryOfLoanSuspension#980] [Ideas]
最近找了好多人。开发商的，信托的，银行的，政府城投平台的。

开发商的人说其实他们真的没钱，他们开发项目都是前期找银行借钱，把钱给政府换地。换了地再借钱，用这些钱盖房子。然后卖出去拿到钱。还债，结算各种成本，剩下利润再去当下一个项目的启动资金。  以前高杠杆的时候，就是这个项目还没完，就把钱弄去下一个项目了。这就是现在很多项目没钱的原因。 现在就是杀了开发商，他也没钱。

信托和银行的人说他们是借钱给开发商/城投做开发贷或者借钱给买房人做购房贷款，收利息。他们也知道很多城投公司都很垃圾（说是最垃圾的公司），但是城投后面有政府啊，他们相信政府会兜底的（虽然他们也不知道政府哪来的钱，但是就是相信政府会兜底）。在以前，不管是开发贷，还是房贷，还是城投贷，信托银行都是稳赚的。

城投的人说他们是给政府做项目（很多项目注定是亏的），帮政府融资借钱的。很多项目是谁都知道的垃圾，但是就是要上的。城投上这些项目，养活一大群关系户，顺便建成了些基础设施。。。

跟这帮人聊了一圈，我发现，其实我们买房的钱开发商就是过个手，多数是进了政府（土地收入和一大堆税费）和银行信托（高额利息）的口袋了。现在这个局面，开发商高杠杆是直接因素，背后政府（城投平台）和银行信托使劲吸血是根本因素。

解决烂尾楼所需资金，肯定是前面谁赚钱了谁现在要出钱最合理。政府话题太大，简单看咬死金融机构确实是个好主意。银行信托过去赚了那么多钱，都是趴在买房人头上吸血的，这不公平。而且现在他们手头钱最多，只有他们能掏钱出来。

最后如果要说城投搞了基建了，政府拿钱是合理的。那基建是所有城市居民都收益的，凭啥让最后买房的人出全部成本？要给基建卖单，麻烦全部纳税人卖单。况且这些基建里多少钱换成了真的城市设施，多少钱换成了关系户的茅台奔驰豪宅都说不清楚。

**Comment by Ventery** [comment]
中国特色社会主义👍👍👍👍👍

**Reply by frshgrass** [reply]
其实这就是事实：我们30年的贷款，以及现在地方政府的天量地方债，七成左右变成了各种三四线城市的各种高楼和云贵各种高速大桥，两成左右被地方实权领导及其家属贪污，其他一成就是各种开发商和被层层七包八包的包工头拿掉。所以在座各位的30年房贷，除了变成砖头外就是地方上岁月静好的领导家庭以及围绕他们的关系户。记住，这部分没有变成砖头的钱还能拿到的钱，就在这些人手里。以及你每天可以看到他们在现实生活以及朋友圈炫耀高端消费，就是用的你们每个月辛苦还贷的上供30年辛苦钱。

**Comment by pinerose99** [comment]
开发商可没少赚钱，看看那些一个个私人飞机。以前都是中国的首富系列。现在就是在推卸责任。

**Reply by kajsnlk** [reply]
以前赚的钱分红给大股东老板了，剩下烂摊子给债主

**Reply by cc2218** [reply]
开发商的人跟我说。就算他们以前赚钱的时候，一年的利润也就利息支出的一半。跟土地支出和税费就更没法比了。

**Reply by zxcs001** [reply]
私人飞机算个屁啊，某大大想要飞机的话，成飞给他造几十个编队都不在话下。

你是真傻还是装傻？资本家就是某些人的狗，白手套，拿着肥肉过一遍手上沾了些油罢了。真的大头谁拿了，你别装外宾了。

**Comment by kajsnlk** [comment]
分税制改革后，地方政府没有多少税源，然后土地财政，搞了房地产庞氏。再说，现在，出生人口下跌这么多了，是个人都知道房地产不可持续，为什么还不转向房地产税，说白了，官僚裙带阶层有大量的房子，他们怎么会向自己开刀。

**Comment by xiejun946** [comment]
这个逻辑显而易见，大家都知道预售制度怎么玩的，目前是政府不管事。就像生孩子一样，你逼不了政府来管，政府也逼不了我们生孩子。

**Reply by zxcs001** [reply]
天真得很，真到紧急关头的情况下，你说不生就不生了？

不生？简单，把你的血都吸干，拿来补贴生的。收税还不容易？银行卡别用了，里面一分钱都别想取，直接划归国库

**Comment by github-huangxin** [comment]
给开发商洗地就大可不必了.

**Comment by Dreaming-Pepper** [comment]
按这样说来，等等说大兴基建，百废待兴。出钱做诱饵，两大家族跟着出钱
